src/html/parser.ts

```diff
@@ -3,24 +3,24 @@ import * as parse5 from 'parse5';
 import { Parser, IAddMessageCallback, IParseOptions } from '../parser';
 import { IMessage } from '../builder';
 
-export type Node = parse5.AST.Default.Node;
-export type TextNode = parse5.AST.Default.TextNode;
-export type Element = parse5.AST.Default.Element;
+export type Node = parse5.Node;
+export type TextNode = parse5.TextNode;
```

Comment:
I think it would be better to use the `DefaultTree*` type for these two as well.

Reply:
Done 👍